packages/node_modules/@webex/internal-plugin-conversation/test/unit/spec/share-activity.js

```diff
@@ -74,5 +76,36 @@ describe('plugin-conversation', () => {
         assert.equal(sa._determineContentCategory(items), 'documents');
       });
     });
+    describe('#upload', () => {
+      let webex;
+      const fakeURL = `${process.env.ENCRYPTION_SERVICE_URL || 'https://encryption-a.wbx2.com'}/encryption/api/v1/keys/8a7d3d78-ce75-48aa-a943-2e8acf63fbc9`;
+
+      before(() => {
+        webex = new MockWebex({
+          upload: sinon.stub().returns(Promise.resolve({body: {downloadUrl: fakeURL}}))
+        });
+      });
+
+      it('checks whether filehash is sent in body while making a call to /finish API', () => {
+        const spy = sinon.spy(webex.upload);
+        const inputData = {
+          phases: {
+            initialize: {
+              fileSize: 3333
+            },
+            finalize: {
+              body: {
+                fileSize: 3333,
+                fileHash: '4ea5c508a6566e76240543f8feb06fd457777be39549c4016436afda65d2330e'
```

Comment:
Where does this file hash come from? Could we make it a variable and show where it originated from?

Reply:
The file hash comes from an encrypted fake URL.
We generate that hash through SHA 256 encryption function.
`const fileHash = sha256(file).toString;`